Atomic/trust.py

```diff
@@ -486,17 +487,20 @@ def get_gpg_id(self, keys):
         """
         if not keys:
             return ""
-        keylist = None
+        (keylist, tmpkey) = None, None
         for key in keys:
             if not os.path.exists(key):
-                # extend when we support parsing inline 'keyData' base64 encoded key
-                return ""
-            else:
-                cmd = ["gpg2", "--with-colons", key]
+                with tempfile.NamedTemporaryFile(delete=False, dir="/run/") as tmpkey:
```

Comment:
Add delete=True and then when the file is closed it should get deleted.  Then you should not need to do code below.  I think?